packages/cli/src/commands/experimental/templates/realtime.ts.template

```diff
@@ -1,14 +1,43 @@
-import { createPubSub, InMemoryLiveQueryStore } from '@redwoodjs/graphql-server'
+import { Redis } from 'ioredis'
 
-export const liveQueryStore = new InMemoryLiveQueryStore()
+import { RedwoodRealtimeOptions } from '@redwoodjs/graphql-server'
 
-// Import each subscription type from 'src/subscriptions/**/*'
-// Note: If you did not setup examples, please replace with your current subscriptions
-import type { NewMessageChannel } from 'src/subscriptions/newMessage/newMessage'
+import subscriptions from 'src/subscriptions/**/*.{js,ts}'
 
-// then use merge the types by intersecting each like: `NewMessageChannel & OtherChannel`
-export type PubSubChannels = NewMessageChannel
+export const publishClient = new Redis()
+export const subscribeClient = new Redis()
```

Comment:
Do these need to be exported? I could not see them being imported from anywhere? 

Also can we have these commented out (and comment out the redis import) by default? The user can uncomment this if they wish to use redis? 

If I do all the setup and then run `yarn rw dev` I get a load of errors because thesse `new Redis()` expect a local redis server to be running. It feels natural to start with the in-memory and then move to redis?

Reply:
Good point. Defaults to in-memory and comment out to start.